### ISSUE TITLE
artwork issue when playing via service

### ISSUE THREAD

**Issue by zala143** [music-assistant/hass-music-assistant#446]
Having an issue displaying artwork on a regular HA dashboard card (custom:mini-media-player) when playing music in a particular scenario.

First off, this is such great project, thank you..

I have a good working solution, with a few media player types tested/working - when using the standard Music Assistant interface to play music, all is working; also displays media artwork on regular HA dashboard cards (custom:mini-media-player).

However, if I play media to the media_player.mass_xxx entity via a service (example below) then the music plays just fine but the artwork never shows on the HA dashboard card.

I would love to solve this issue and hopefully its something I'm not doing correctly - my use case is that that example service below is being called by a Music Tag scanning blueprint to play music when NFC tags are scanned - works great except for the artwork in played via a media_player.mass_xxx entity.  

Thanks in advance.

Example Service 
```
service: media_player.play_media
data:
  media_content_id: https://open.spotify.com/album/6DN7GcZF1HywzrkGN6Eeqk
  media_content_type: music
  enqueue: play
target:
  entity_id: media_player.mass_up2stream_01
```

**Comment by marcelveldt** [comment]
Does the artwork work on the default HA card ?

**Comment by zala143** [comment]
ok, testing with regular internal HS media card 

```
type: media-control
entity: media_player.mass_up2stream_01

```
playing same media via the service (yaml code posted in my original message) - plays fine but doesn't show the artwork. 

<img width="501" alt="via_Service" src="https://user-images.githubusercontent.com/20558520/174489581-297744b3-33e8-4751-baf5-57207ff8c05c.png">

playing same track via Music Assistant shows artwork. 
<img width="508" alt="via_MusicAssistant" src="https://user-images.githubusercontent.com/20558520/174489584-d80082a9-8ce9-43ff-acdd-671b1a236abe.png">

strange..

**Comment by zala143** [comment]
no sure if there's another way to solve what I'm trying to do - is there a way to invoke Music Assistant to play an album, artist or playlist via a service - so rather then me browsing the MA UI to discover content- could I simply invoke my selection via yaml script? I think i can then rework the Tag Music player to use your service/script rather than the internal HA one that seems to have an issue with artwork in this mode?

your Music Assistant is freaking awesome, just trying to use it to its full effect - thanks

**Comment by marcelveldt** [comment]
That is a strange bug. So if you play the Spotify URL/URL directly the artwork is missing.
Leave this open as bug so I can have look, I'm pretty sure it's just something very small.

**Comment by marcelveldt** [comment]
fixed in 2022.6.3, please reopen if you still experience any issues, thanks!
